org.mixedrealitytoolkit.spatialmanipulation/ObjectManipulator/ObjectManipulator.cs

```diff
@@ -161,6 +161,32 @@ public InteractionFlags AllowedInteractionTypes
             set => allowedInteractionTypes = value;
         }
 
+        [SerializeField]
+        [Tooltip("The movement type of the rigid body during manipulation.")]
+        private MovementType rigidbodyMovementType = MovementType.VelocityTracking;
```

Comment:
Since this value can be overridden by an interactor, I think we should rename `rigidbodyMovementType` so to indicate that this value might not used.  

Perhaps `defaultRigidbodyMovementType`  or `preferredRigidbodyMovementType`

Reply:
My personal take is that isn't necessary in order to be clear.  For example, the XRI XRGrabInteractable's property is just named "movementType", though it is understood to be potentially overridden by an interactor without expressing that in the name.

In our case, at least, I had added a description of the overriding behavior in the <remarks> section of the documentation on the RigidbodyMovementType property.  Would that be sufficient?

Reply:
Ok :)  

Can you all add something to the tooltip?  So developers in the inspector have some helpful documentations as well.

example:

```cs
[Tooltip("The preferred movement type of the rigid body during manipulation. Selecting interactors may override this and define a different movement type.")]
```

Reply:
Done.  Also, since this involves new API, updated the minor version of the spatial manipulation package.